src/app/containers/FeaturesAnalysis/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,317 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CpsRelatedContent should render Story Feature components in Live environment 1`] = `
+<DocumentFragment>
+  <section
+    aria-labelledby="related-content-heading"
```

Comment:
incorrect name for this "related-content-heading", should be "features-analysis-heading"?

Reply:
I see you've update it now.... thanks